assets/js/state/hosts.js

```diff
@@ -71,6 +87,8 @@ export const hostsListSlice = createSlice({
   },
 });
 
+export const DEREGISTER_HOST = 'DEREGISTER_HOST';
```

Comment:
We need a new name for this, the most difficult thing as always XD hehe
`CHECK_HOST_IS_DEREGISTARABLE` maybe?
And the next would be:
`CANCEL_CHECK_HOST_IS_DEREGISTARABLE_ACTION` or similar.

Suggest alternatives please XD